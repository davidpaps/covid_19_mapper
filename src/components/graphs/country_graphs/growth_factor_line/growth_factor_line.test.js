import * as React from "react";
import { shallow } from "enzyme";
import GrowthFactorLine from "./growth_factor_line";

describe("Country Growth Factor", () => {
  let wrapper;

  beforeEach(
    () =>
      (wrapper = shallow(
        <GrowthFactorLine createLineLabels={[]} data={[]} country={[]} />
      ))
  );

  it("should render a <div />", () => {
    expect(wrapper.find("div").length).toEqual(3);
  });

  it("should render a <Line />", () => {
    expect(wrapper.find("Line").length).toEqual(1);
  });

  it("renders correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
