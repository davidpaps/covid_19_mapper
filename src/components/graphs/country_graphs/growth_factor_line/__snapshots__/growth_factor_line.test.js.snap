// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Country Growth Factor renders correctly 1`] = `ShallowWrapper {}`;
